### DISCUSSION TITLE
Is YimMenu safe?

### DISCUSSION THREAD

**Discussion by Charlie234563** [YimMenu/YimMenu#3504] [Q&A] [answered]
I want to download the YimMenu but I'm too sceptical of getting a virus as when i put the file through virus total it comes back with 18out of 75 malicious entities

**Comment by lonelybud** [comment]
duplicate https://github.com/YimMenu/YimMenu/discussions/2988

**Comment by Rxann** [comment]
It uses concepts that malware use(it is not malware) but the heuristics register it as malware from these concepts.

**Comment by FIying-Scotsman** [comment, marked as answer]
Yes and as long as you got the dll from this repo you'll be fine. If you got it from a YT video or dodgy site then no, we can't verify **_THAT_** download was safe when the ones on this repo are.

**Reply by Charlie234563** [reply]
alr thanks for the help
